### PR TITLE
W2D1 curriculum specialist & day lead updates

### PR DESCRIPTION
**Pull request by BrodiePearson**
This PR replaces last year's W2D1 tutorials with this years updated set of 4 tutorials.

Checked with Python 3.9 locally

### CODE REVIEW

File: tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial1.ipynb [hunk 1]
```diff
@@ -314,293 +396,216 @@
...
...HMCvL5PMceeyybNm3iuOOOc1yuO+64AwDDMP5/O3cPklwYhnH8Ck2CaGjIlqytHBz6HIJwiIKKir6lpraCKGhIXAKbQoKgqYagiEAIyz6XmgKnChJyCpKaDRqiCCN6pw6vfYC82OsR/r/Fg/e5j/fjdul5jqqrqxWLxbS0tKTJyUnl5eVJki4vL9Xa2qqXlxfF43HZ7Xbjmvv7++rs7JTNZjM+K5FIqKOjQxcXFzo9PZXb7TZqfr9fc3NzkqRAICCv1/u/vgIAAL7FT4YAAFOwWq1aXV2VxWLR2dmZioqKtLy8bNQPDg4Ui8U0PDysqakpI7BJUk1NjWZnZ/X09KTNzc2Ua/b19aUENkkqKSnR/Py8JGlvb+/beVwul2ZmZjK5RAAA/ok12wMAAPChrq5Ovb29CoVCGhsbU1lZmVE7OTmRJPX09Hzb29TUJEk6Pz//UotGozo+Ptbd3Z2en5/1/v6ux8dHSTL20H3W1dWVEgwBAMgWQhsAwFQKCwtTXj/c3t5Kkjwejzwez4/99/f3xnEymdTo6KiCweCP53+Et8/Ky8vTHRkAgF9FaAMA5IS3tzdJUnt7u7Fn7TtOp9M4XlxcVDAYlMvl0sLCgmpra1VcXKz8/HxdX1+rqqpKP23tLigoyOwCAAD4R4Q2AEBO+LhVcnx8XN3d3Wn1hMNhSTKC29/i8XhmBwQA4JfwIBIAQE5oaWmRJO3u7qbd8/DwIElyOBxfaltbWxmZCwCA30ZoAwDkhIGBATmdTq2vrysQCOj19TWlnkwmtbOzo6urK+O9yspKSdLKykrKuaFQSBsbG78/NAAAGUBoAwDkBKvVqnA4LIfDIZ/Pp4qKCrW1tWloaEiNjY0qLS1Vf3+/bm5ujB6v1yuLxSKfz6f6+nqNjIyooaFBg4ODmp6ezuJqAABIH6ENAJAznE6notGo/H6/7Ha7IpGIjo6OlEgk5Ha7tba2ZtxGKUlut1uRSETNzc2Kx+M6PDyUzWbT9va2JiYmsrgSAADSl/f+02OzAAAAAABZxz9tAAAAAGBihDYAAAAAMDFCGwAAAACYGKENAAAAAEyM0AYAAAAAJkZoAwAAAAATI7QBAAAAgIkR2gAAAADAxAhtAAAAAGBihDYAAAAAMDFCGwAAAACYGKENAAAAAEyM0AYAAAAAJvYH6w1ZIRhT7WcAAAAASUVORK5CYII=",
```
Comment (iamzoltan) [inline tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial1.ipynb:588]: This image should not be here.

File: tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial1.ipynb [hunk 2]
```diff
@@ -952,7 +796,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.12.2"
   }
  },
  "nbformat": 4,
```
Comment (iamzoltan) [inline tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial1.ipynb:802]: looks like you have the wrong python version.

File: tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial2.ipynb [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "execution": {}
    },
    "source": [
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neuromatch/climate-course-content/blob/main/tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial2.ipynb) &nbsp; <a href=\"https://kaggle.com/kernels/welcome?src=https://raw.githubusercontent.com/neuromatch/climate-course-content/main/tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial2.ipynb\" target=\"_parent\"><img src=\"https://kaggle.com/static/images/open-in-kaggle.svg\" alt=\"Open in Kaggle\"/></a>"
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neuromatch/climate-course-content/blob/main/tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial3.ipynb) &nbsp; <a href=\"https://kaggle.com/kernels/welcome?src=https://raw.githubusercontent.com/neuromatch/climate-course-content/main/tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial3.ipynb\" target=\"_parent\"><img src=\"https://kaggle.com/static/images/open-in-kaggle.svg\" alt=\"Open in Kaggle\"/></a>"
```
Comment (iamzoltan) [inline tutorials/W2D1_FutureClimate-IPCCIPhysicalBasis/W2D1_Tutorial2.ipynb:9]: again, these tutorial links should reflect the names of the files themselves

**Review by iamzoltan — CHANGES_REQUESTED** [review]
Nice Updates! There are somethings wrong your with PR. We need to address them before we can merge.

Todo:

- [ ]  Before saving, clear all outputs and restart your kernel.
- [ ]  Make sure to use a python environment with py3.9
- [ ]  Naming consistency. Your kaggle/colab links should point to the tutorials they are embedded in
- [ ]  Your tutorials days must also be consistent, increasing starting from 1. 
- [ ]  You should change your materials file to reflect the changes in the number of tutorials for that day

Please add those changes to your PR.

Additionally, please add me as production editor to the tutorials.

Let me know if you have any questions.

**Comment by BrodiePearson** [comment]
@iamzoltan I have updated the PR to fix all the issues you mentioned. I won't be able to address any other significant changes until next week.

**Pull request closed without merging**